eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshHTTPServer.java

```diff
@@ -116,118 +116,90 @@ public class EventMeshHTTPServer extends AbstractHTTPServer {
+            "eventMesh-clientManage", true);
 
         adminExecutor = ThreadPoolFactory.createThreadPoolExecutor(
-                eventMeshHttpConfiguration.getEventMeshServerAdminThreadNum(),
-                eventMeshHttpConfiguration.getEventMeshServerAdminThreadNum(),
-                new LinkedBlockingQueue<>(50), "eventMesh-admin",
-                true);
+            eventMeshHttpConfiguration.getEventMeshServerAdminThreadNum(),
+            eventMeshHttpConfiguration.getEventMeshServerAdminThreadNum(),
+            new LinkedBlockingQueue<>(50), "eventMesh-admin",
+            true);
 
         replyMsgExecutor = ThreadPoolFactory.createThreadPoolExecutor(
-                eventMeshHttpConfiguration.getEventMeshServerReplyMsgThreadNum(),
-                eventMeshHttpConfiguration.getEventMeshServerReplyMsgThreadNum(),
-                new LinkedBlockingQueue<>(100),
-                "eventMesh-replyMsg", true);
-    }
-
-    public ThreadPoolExecutor getBatchMsgExecutor() {
-        return batchMsgExecutor;
-    }
-
-    public ThreadPoolExecutor getSendMsgExecutor() {
-        return sendMsgExecutor;
-    }
-
-    public ThreadPoolExecutor getReplyMsgExecutor() {
-        return replyMsgExecutor;
+            eventMeshHttpConfiguration.getEventMeshServerReplyMsgThreadNum(),
+            eventMeshHttpConfiguration.getEventMeshServerReplyMsgThreadNum(),
+            new LinkedBlockingQueue<>(100),
+            "eventMesh-replyMsg", true);
     }
 
-    public ThreadPoolExecutor getPushMsgExecutor() {
-        return pushMsgExecutor;
-    }
-
-    public ThreadPoolExecutor getClientManageExecutor() {
-        return clientManageExecutor;
-    }
-
-    public ThreadPoolExecutor getAdminExecutor() {
-        return adminExecutor;
-    }
-
-    public RateLimiter getMsgRateLimiter() {
-        return msgRateLimiter;
-    }
-
-    public RateLimiter getBatchRateLimiter() {
-        return batchRateLimiter;
-    }
-
-    public Registry getRegistry() {
-        return registry;
+    public void shutdownThreadPool() {
+        if (batchMsgExecutor != null) {
+            batchMsgExecutor.shutdown();
+        }
+        if (adminExecutor != null) {
+            adminExecutor.shutdown();
+        }
+        if (clientManageExecutor != null) {
+            clientManageExecutor.shutdown();
+        }
+        if (sendMsgExecutor != null) {
+            sendMsgExecutor.shutdown();
+        }
+        if (remoteMsgExecutor != null) {
+            remoteMsgExecutor.shutdown();
+        }
+        if (pushMsgExecutor != null) {
+            pushMsgExecutor.shutdown();
+        }
+        if (replyMsgExecutor != null) {
+            replyMsgExecutor.shutdown();
+        }
     }
 
     private void init() throws Exception {
         if (log.isInfoEnabled()) {
             log.info("==================EventMeshHTTPServer Initialing==================");
         }
-        super.init("eventMesh-http");
+        super.init("EventMesh-http");
```

Comment:
eventMesh or EventMesh? Can be consistent.

Reply:
get it.